plans/subplans/provision.pp

```diff
@@ -138,23 +161,27 @@
 
   # Constructs a tfvars file to be used by Terraform
   $tfvars = epp('pecdm/tfvars.epp', {
-    project              => $project,
-    user                 => $ssh_user,
-    lb_ip_mode           => $lb_ip_mode,
-    ssh_key              => $ssh_pub_key_file,
-    region               => $cloud_region,
-    node_count           => $node_count,
-    instance_image       => $_instance_image,
-    subnet               => $subnet,
-    subnet_project       => $subnet_project,
-    firewall_allow       => $firewall_allow,
-    architecture         => $architecture,
-    cluster_profile      => $cluster_profile,
-    replica              => $replica,
-    compiler_count       => $compiler_count,
-    disable_lb           => $disable_lb,
-    provider             => $provider,
-    extra_terraform_vars => $extra_terraform_vars
+    project                => $project,
+    user                   => $ssh_user,
+    windows_user           => $windows_user,
+    lb_ip_mode             => $lb_ip_mode,
+    ssh_key                => $ssh_pub_key_file,
+    region                 => $cloud_region,
+    node_count             => $node_count,
+    instance_image         => $_instance_image,
+    windows_node_count     => $windows_node_count,
+    windows_instance_image => $_windows_instance_image,
+    windows_password       => $windows_password.unwrap,
```

Comment:
If these are sent as undef on non-azure platforms then the template will not include them. That is best because Terraform complains if you send it variables which are not defined inputs